src/js/utils/constant.js

```diff
@@ -0,0 +1,9 @@
+export { EMAIL_REGEX, USER_EMAIL, USER_PASSWORD, PASSWORD_REGEX };
+
+const EMAIL_REGEX =
+  /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$/i;
+
+const USER_EMAIL = "test@codeit.com";
+const USER_PASSWORD = "codeit101";
+
+const PASSWORD_REGEX = /^(?=.*[a-zA-Z])(?=.*[0-9]).{8,25}$/;
```

Comment:
@gaem-gaem 

[제안]

export 는 보통 맨 아래에 위치합니다.

```suggestion
const EMAIL_REGEX =
  /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$/i;

const USER_EMAIL = "test@codeit.com";
const USER_PASSWORD = "codeit101";

const PASSWORD_REGEX = /^(?=.*[a-zA-Z])(?=.*[0-9]).{8,25}$/;

export { EMAIL_REGEX, USER_EMAIL, USER_PASSWORD, PASSWORD_REGEX };
```

혹은 각 변수마다 export 해도 됩니다.

```suggestion
export const EMAIL_REGEX =
  /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$/i;

export const USER_EMAIL = "test@codeit.com";
export const USER_PASSWORD = "codeit101";

export const PASSWORD_REGEX = /^(?=.*[a-zA-Z])(?=.*[0-9]).{8,25}$/;
```